tests/conftest.py

```diff
@@ -47,6 +48,50 @@ def rocket(solid_motor):
     return example_rocket
 
 
+@pytest.fixture
+def m():
+    return numericalunits.m
+
+
+@pytest.fixture
+def kg():
+    return numericalunits.kg
```

Comment:
Should there by a fixture for seconds as well?

```suggestion
@pytest.fixture
def m():
    return numericalunits.m


@pytest.fixture
def kg():
    return numericalunits.kg


@pytest.fixture
def s():
    return numericalunits.s
```

Reply:
Sure thing! But I think we use input based on time only on addParachutes method (the lag input) so in that case I think I agree using only mass and length for now